crates/cribo/debug_resolver.py

```
@@ -0,0 +1,16 @@
+import sys
```

Pylint (3.3.7)

[convention] Missing module docstring

(C0114)